### PR TITLE
Fix dialog text

### PR DESCRIPTION
**Pull request by MrTheGood**
Changes:
 * Fix dialog text

### CODE REVIEW

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -52,4 +52,7 @@
     <string name="error_could_not_set_current_location">Kon de memory locatie niet op de huidige locatie zetten.</string>
     <string name="warn_enable_permissions">Sta locatie toe</string>
     <string name="warn_enable_permissions_message">WHIB moet je locatie weten om memories op je locatie op te slaan!</string>
+    <string name="string_delete_memory">Verwijder Herinnering</string>
+    <string name="string_delete_memory_description">Weet je zeker dat je de herinnering wilt verwijderen? Dit kan niet ongedaan worden</string>
```
Comment (Hicham010) [inline app/src/main/res/values-nl/strings.xml]: Maak de zin formeel

Comment (Hicham010) [inline app/src/main/res/values-nl/strings.xml]: Maak de zin formeel. We doen niet aan tutoyeren.

Comment (MrTheGood) [reply app/src/main/res/values-nl/strings.xml]: done.

**Review by Hicham010 — APPROVED** [review]
Nice work

**Review by Almightywesley — APPROVED** [review]
commit abuse will the next time not be accepted

**Comment by MrTheGood** [comment]
What do you mean "commit abuse"? I saw a problem, and I fixed the problem. 

It was the only text not translated to Dutch, and now it is.

**Pull request merged**